docs/apis/GTFS-GraphQL-API.md

```diff
@@ -0,0 +1,44 @@
+# GTFS GraphQL API
+
+The GTFS GraphQL API was created for the Digitransit project and is used heavily by
+[digitransit-ui](https://github.com/HSLdevcom/digitransit-ui).
+
+[otp-react-redux](https://github.com/opentripplanner/otp-react-redux) has also migrated to this API
+in 2023.
+
+## URLs 
+ - GraphQL endpoint: `http://localhost:8080/otp/routers/default/index/graphql`
```

Comment:
I've wondered even before if it would make sense to make an API endpoint a link in docs just so it's easier to copy from a browser or would it just be confusing as opening the link doesn't open a proper page on browser.

Reply:
I didn't make it a link because clicking on it doesn't lead to any useable page. You're supposed to copy & paste it somewhere.